src/pdn/test/tinyRocket.api.ok

```diff
@@ -20,7 +20,7 @@
 [INFO PDN-0011] ****** INFO ******
 Type: stdcell, grid
     Stdcell Rails
-      Layer: metal1  -  width: 0.170 
+      Layer: metal1  -  width: 0.170  pitch: 2.800 
```

Comment:
Shouldn't the printout for the macro indicate if the grid is just over the pins?

Reply:
Yes, I will update for this.

Reply:
Done